src/resources/views/crud/columns/upload_multiple.blade.php

```diff
@@ -6,12 +6,15 @@
     $column['escaped'] = $column['escaped'] ?? true;
     $column['wrapper']['element'] = $column['wrapper']['element'] ?? 'a';
     $column['wrapper']['target'] = $column['wrapper']['target'] ?? '_blank';
-    $column_wrapper_href = $column['wrapper']['href'] ?? function($file_path, $disk, $prefix) { return 
-        !is_null($disk) ?
-        (isset($column['temporary']) ?
-            asset(\Storage::disk($disk)->temporaryUrl($file_path, Carbon\Carbon::now()->addMinutes($column['temporary']))) :
-            asset(\Storage::disk($disk)->url($file_path))
-        ) : asset($prefix.$file_path)); 
+    $column_wrapper_href = $column['wrapper']['href'] ?? 
+    function($file_path, $disk, $prefix) { 
+        if (is_null($disk)) {
+            return $prefix.$file_path;
+        }
+        if (isset($column['temporary'])) {
+            return asset(\Storage::disk($disk)->temporaryUrl($file_path, Carbon\Carbon::now()->addMinutes($column['temporary'])));
+        }
+        return asset(\Storage::disk($disk)->url($file_path));
```

Comment:
I wish we could remove the `{` & `}` in the conditionals here... it would still work and it would be so much cleaner... but I'm afraid the bracket police will come for us 👀😅

Reply:
I would be one of those. Brackets are probably the most valuable item in terms of readability. 

I agreed that chained conditionals are difficult to understand, but I wouldn't went that far to convert simple conditionals into if conditions. 

```php
condition ?
do 1 :
do 2;

if (condition) {
do1;
}
do2;
```

Whatever, it's fine anyway to me. 👍